tests/virt/cluster/common_templates/fedora/test_fedora_os_support.py

```diff
@@ -225,6 +225,7 @@ def test_pause_unpause_vm(self, golden_image_vm_object_from_template_multi_fedor
             vm=golden_image_vm_object_from_template_multi_fedora_os_multi_storage_scope_class,
         )
 
+    @pytest.mark.gating
```

Comment:
same question - we have rhel, why we also interested in fedora as part of gating?